### PR TITLE
Adding new advert

### PR DESCRIPTION
**Pull request by Ocieplino**
First part done, now you can display new MUSICAN advert form through button on sidebar, then it asks which advert you want to create, only musician is aviable for now. Then it displays form that in future should be send to backend server.

### CODE REVIEW

**Review by msciwoj23 — APPROVED** [review]

File: front/src/App.js [hunk 1]
```diff
@@ -28,6 +29,11 @@ class App extends Component {
                 )}
               </Col>
               <Col>
+                {localStorage.getItem("muzi_muzi_token") !== null ? (
+                  <Route path="/new-advert" component={AddMusicianAdvert} />
+                ) : (
+                  true
+                )}
                 <Route exact path="/" component={News} />
```
Comment (sznowak13) [inline front/src/App.js:48]: Maybe put all the secured routes to a function that returns them al when user is logged in?

File: front/src/mainContent/AddMusicianAdvert.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              data={professions}
+              placeholder="Select Profession"
+              style={{ width: 224 }}
+              onOpen={this.handleUpdate}
+              onSearch={this.handleUpdate}
+              renderMenu={menu => {
+                if (professions.length === 0) {
+                  return (
+                    <p
+                      style={{ padding: 4, color: "#999", textAlign: "center" }}
```
Comment (sznowak13) [inline front/src/mainContent/AddMusicianAdvert.js]: Again - start putting those simple logic into functions and reuse them in code.

File: front/src/mainContent/AddMusicianAdvert.js [hunk 3]
```diff
@@ -47,61 +51,107 @@ export default class AddMusicianAdvert extends Component {
...
-              data={professions}
-              placeholder="Select Profession"
-              style={{ width: 224 }}
-              onOpen={this.handleUpdate}
-              onSearch={this.handleUpdate}
-              renderMenu={menu => {
-                if (professions.length === 0) {
-                  return (
-                    <p
-                      style={{ padding: 4, color: "#999", textAlign: "center" }}
```
Comment (pKraszewska) [inline front/src/mainContent/AddMusicianAdvert.js]: maybe move this to css file since it is doubled in this file?

**Review by pKraszewska — COMMENTED** [review]
good job. love the select pickers!!

**Review by sznowak13 — APPROVED** [review]
Very good job, remember about separating logic into functions and put longer commit details in the commit description - add 2 enters after commit title.

**Pull request merged**